qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -733,6 +737,18 @@ def on_scroll_pos_changed(self):
         self._update_window_title('scroll_pos')
         self.widget.update_tab_title(idx, 'scroll_pos')
 
+    @pyqtSlot()
+    def on_audio_changed(self, tab, _muted):
```

Comment:
This should be private (`_on_audio_changed`).

Background: I used to think that slots should always be public, but I changed my mind at some point: Slots not accessed from the outside should still be private.